src/textual/_terminal_features.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import annotations
+from typing import NamedTuple
+
+from textual._ansi_sequences import TERMINAL_MODES_ANSI_SEQUENCES
+from textual._terminal_modes import Mode
+
+
+class TerminalSupportedFeatures(NamedTuple):
+    """
+    Handles information about the features the current terminal emulator seems to support.
+    """
+
+    mode2026_synchronized_update: bool = False
```

Comment:
Correct me if I'm wrong, but it's not officially know as "mode 2026" which is rather cryptic.

Perhaps "synchronised_output" would be lest cryptic?